src/ts/demofactory.ts

```diff
@@ -90,11 +110,13 @@ export namespace DemoFactory {
             new AirPlayToggleButton(),
             new CastToggleButton(),
             new VRToggleButton(),
-            new AudioTrackSelectBox({
-              cssClasses: ['my-audio-track-selection-demo'],
+            new SettingsToggleButton({
+              settingsPanel: audioTrackSettingsPanel,
+              cssClass: 'ui-audiotrack-setting-toggle-button',
```

Comment:
The `cssClass` of the `SettingsToggleButton` is `ui-settingstogglebutton`, so this should be something along `ui-audiotracksettingstogglebutton` (the pattern is always to extend the parent's name).